src/segmantic/seg/losses.py

```diff
@@ -0,0 +1,110 @@
+from __future__ import annotations
+
+import torch
+from monai.networks import one_hot
+from monai.utils import LossReduction
+from torch.nn.modules.loss import _Loss
+
+
+class AsymmetricUnifiedFocalLoss(_Loss):
```

Comment:
this should be AsymmetricFocalLoss:

https://github.com/mlyg/unified-focal-loss/issues/8#issuecomment-1002105347

to compute AsymmetricUnifiedFocalLoss we also need the AsymmetricTverskyLoss